### DISCUSSION TITLE
Delight: Curate a list of rooms for new users that help to understand the explore functionality

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#946] [Product Feedback]
Today we ask users to join a room and show the Explore Rooms model. It can be overwhelming and confusing if you're new to Element and Matrix, many users do not know where to start or how to get involved. 

According to the Spec we must order rooms by size but how else might we get users involved in some rooms or spaces, educating them on the community side of matrix, without overwhelming.

**Comment by t3chguy** [comment]
Curating should really happen on the server given Element can connect to any server and has no way to know which rooms are accessible over federation until it attempts to join them

**Comment by t3chguy** [comment]
The Matrix spec also says that /publicRooms has to be sorted by number of users descending so will also need spec changes

**Comment by turt2live** [comment]
Related: the spec's plans to replace the room directory with a space.

**Comment by germain-gg** [comment]
more-so than curating, it would be great that a homeserver admin have the ability to make some new users automatically join rooms or spaces.

This would be extremely useful for all the workplace segment, and maybe for the community segments too (for them to see the code of conduct in an announcement-only room for example)

**Comment by t3chguy** [comment]
A homeserver admin already has that power, synapse has had this feature for a long time and ems makes use of it

**Comment by niquewoodhouse** [comment]
Why does a home server admin get that right and a space admin doesn't?

If a space admin can trick you into joining 100 bad rooms when joining a space, can the same not be said for a home server admin?

**Comment by t3chguy** [comment]
A homeserver admin gets full control of your account, that's just how Matrix works

**Comment by daniellekirkwood** [comment]
Moving to discussions as is the process
